example/src/messages.js

```diff
@@ -16,8 +15,30 @@ const messages = {
     "header.search": "Search",
     "header.search.placeholder": "Find what you're looking for",
     "header.search.hint": "Search any products and/or materials",
-    "header.search.results": "Results"
+    "header.search.results": "Results",
+
+    "cart.header.advice": "You're ${label} away from free shipping",
```

Comment:
Is this available if I just import the Cart component?

Reply:
I think if we have react-intl in the other repository too things are going to mix up

Reply:
NO it isn't. It belongs to the example project. We still need to add texts in the project that use it